xsd/netex_framework/netex_genericFramework/netex_path_version.xsd

```diff
@@ -445,11 +445,6 @@ The same PATH LINK may occur in different sequences in different NAVIGATION PATH
 		</xsd:annotation>
 		<xsd:sequence>
 			<xsd:element ref="GenericPathLinkRef"/>
-			<xsd:element name="Description" type="MultilingualString" minOccurs="0">
```

Comment:
That's a compatibility break ... but I personally can accept this one (the 2 "Description" are just 1 element away !)